docs/source/cli.rst

```diff
@@ -31,7 +31,7 @@ Read it
     optional arguments:
       -h, --help            show this help message and exit
       --pattern PATH, -p PATH
-                            paths to pattern definition files
+                            paths to pattern definition files or modules
```

Comment:
This should no longer say that it is a path at all, since now it's just any importable module.